pycbc/bank/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+This packages contains modules for dealing with the template bank
+"""
+
+
+from .conversions import *
```

Comment:
What is the point of this import?

Reply:
I think this means that you can do "from pycbc import bank" and then use "bank.conversions" - however I've never fully got the knack of this kind of thing